pilot/pkg/networking/core/v1alpha3/loadbalancer/loadbalancer.go

```diff
@@ -80,6 +81,9 @@ func ApplyLocalityLBSetting(
 			return
 		}
 		applyLocalityFailover(locality, loadAssignment, localityLB.Failover)
+	} else if !enableFailover && (len(localityLB.FailoverPriority) > 0 || len(localityLB.Failover) > 0) {
+		// Warning for DestinationRule occurs during validation, but meshConfig.localityLbSettings could still be missed here 
+		log.Warnf("failover configuration not applied due to missing outlier detection")
```

Comment:
isn't this the default case for all users? so won't it spam a ton?

Reply:
Oh I guess this is for users that explicitly _customize_ the behavior as well. I do still worry that a user will customize in mesh config, then this will spam possibly n^2 logs

Reply:
If I am switching everything to warning validation messages, I could potentially remove the log message and follow-up this PR with an istioctl analyzer message

Reply:
removed log message